build.sh

```diff
@@ -8,55 +8,73 @@ mkdir ${SRC}
 mkdir ${BUILD}
 mkdir ${INSTALL}
 root=$(pwd)
-cores=4
-#VERSION_BINUTILS="2.33"
-VERSION_GCC="10.2.0"
+cores=8
+#VERSION_BINUTILS="2.37"
+VERSION_GCC="11.2.0"
 VERSION_LIBC="2.0.0"
 
 # Get sources
-wget -q   "https://dl.bintray.com/osx-cross/avr-patches/avr-binutils-2.33-size.patch" &
-wget -q   "https://dl.bintray.com/osx-cross/avr-patches/avr-libc-${VERSION_LIBC}-atmega168pb.patch" &
-wget -qO- "https://ftp.gnu.org/gnu/binutils/binutils-2.33.1.tar.bz2" | tar xj --directory ${SRC} &
+wget -q   "https://raw.githubusercontent.com/archlinux/svntogit-community/c3efadcb76f4d8b1a3784015e7c472f59dbfa7de/avr-binutils/repos/community-x86_64/avr-size.patch" &
+wget -q   "https://raw.githubusercontent.com/osx-cross/homebrew-avr/d2e2566b06b90355952ed996707a0a1a24673cd3/Patch/avr-libc-add-mcu-atmega168pb.patch" &
+#wget -q   "https://raw.githubusercontent.com/osx-cross/homebrew-avr/18d50ba2a168a3b90a25c96e4bc4c053df77d7dc/Patch/avr-binutils-elf-bfd-gdb-fix.patch" &
```

Comment:
Is there a reason why this patch is commented out?

Reply:
The patch only adds `#include <string.h>`, but that does not seem necessary since we don't get an error.